content/docs/guides/opentelemetry.md

```diff
@@ -16,6 +16,37 @@ To enable the receiver you need to toggle the flag `--web.enable-otlp-receiver`.
 $ prometheus --enable-feature=otlp-write-receiver
 ```
 
+## Send OpenTelemetry Metrics to the Prometheus Server
+
+OpenTelemetry SDKs and instrumentation libraries can be configured via [standard environment variables](https://opentelemetry.io/docs/languages/sdk-configuration/). The following are the OpenTelemetry variables needed to send OpenTelemetry metrics to a Prometheus server on localhost:
+
+```shell
+export OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
+export OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:9090/api/v1/otlp/
+```
+
+Turn off traces and logs, because the Prometheus server supports metrics only:
+
+```shell
+export OTEL_TRACES_EXPORTER=none
+export OTEL_LOGS_EXPORTER=none
+```
```

Comment:
Should we tell people to use [OTEL_EXPORTER_OTLP_METRICS_ENDPOINT](https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/#otel_exporter_otlp_metrics_endpoint) instead? That way you don't need to disable traces/logs I believe